src/brain_teasers/python/array_product/README.md

```diff
@@ -0,0 +1,33 @@
+# Array Product Except Self
```

Comment:
You can change the document title to "Arry Product"  to match the Python file name

Reply:
Thanks. The title is now updated.